docs/release-notes/highlights-1.9.0.asciidoc

```diff
@@ -29,3 +29,9 @@ To provide more functionalities and better user experience we are moving away fr
 [id="{p}-190-elastic-license-v2"]
 ==== Elastic Cloud on Kubernetes moves to Elastic License v2
 Following the Elastic Stack licensing changes in `7.11.0`, ECK `1.9.0` moves to Elastic License v2. For more details, check this link:https://www.elastic.co/blog/elastic-license-v2[blog post].
+
+[float]
+[id="{p}-120-known-issues"]
```

Comment:
This key needs to be unique otherwise the build cannot succeed

Reply:
wasn't aware of this... resolved.